### PR TITLE
[CODEQUALITY] after merging #349, fix code style of whole project

### PR DESCRIPTION
**Pull request by jpmschuler**
After adding all the QA this PR should update the code base to the new style.

This is a WIP which might of course easily break. It needs to be done AFTER https://github.com/T3Voila/templavoilaplus/pull/349 obviously as soon as that is merged I will rebase this to main (currently the base is that PR).

Will rebase and add fixes regularly until everything is as green as possible. However this can of course be merged before everything is green, because still it is a code quality improvement.

### CODE REVIEW

**Review by opi99 — CHANGES_REQUESTED** [review]
Please exclude Classes/Form path, as this are more or less copys and advanced classes from core. The changes would make it harder for me to get them in tree with TYPO3 Core

File: Resources/Private/Language/Backend/PageLayout.xlf [hunk 1]
```diff
@@ -1,121 +1,110 @@
...
-    <file source-language="en" datatype="plaintext" original="messages" date="2014-07-13T20:55:34Z" product-name="templavoilaplus">
-        <header/>
+<?xml version="1.0" encoding="UTF-8"?>
+<xliff version="1.2" xmlns:t3="http://typo3.org/schemas/xliff" xmlns="urn:oasis:names:tc:xliff:document:1.2">
+    <file source-language="en" original="messages" datatype="plaintext" product-name="templavoilaplus" date="2022-05-03T15:47:31+02:00">
+        <header></header>
         <body>
-            <trans-unit id="mlang_tabs_tab" xml:space="preserve">
-                <source>Page</source>
+            <trans-unit id="error.fce.noMapping" resname="error.fce.noMapping">
```
Comment (opi99) [inline Resources/Private/Language/Backend/PageLayout.xlf:6]: Please do not reorder by id ... they are sorted by usage points yet, which may be changed later on.

**Comment by jpmschuler** [comment]
> Please exclude Classes/Form path, as this are more or less copys and advanced classes from core. The changes would make it harder for me to get them in tree with TYPO3 Core
Classes/Form/**/*` should be handled as it is PHP code. The changes are just whitespace or `array()` converted to `[]` So even in a diff can be easily distinguished from real code changes

**Comment by jpmschuler** [comment]
Probably this should then be tackled in split PRs, if at all

**Pull request closed without merging**